src/components/homePage/mainHeader/mainHeader.tsx

```diff
@@ -0,0 +1,66 @@
+import styles from "./mainHeader.module.scss";
+import superman_header from "../../../assets/img/superman_header.jpg";
+import batman_header from "../../../assets/img/batman_header.jpg";
+import batma_header1 from "../../../assets/img/batman_headerCopy.jpg";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+function MainHeader() {
+  const slides = [
```

Comment:
Move slides to separated file and pas it through the props

Reply:
fixed